src/indigo/less/components/inline-edit-input.less

```diff
@@ -10,13 +10,14 @@
     background: transparent;
     padding: 0px 28px 0px 0px;
     border: none;
+    box-shadow: none;
     border-radius: 0px;
     border-bottom: 2px solid @gray-light;
   }
 
   &:hover .form-control,
-  & > .form-control:hover,
-  & > .form-control:focus {
+  & .form-control:hover,
+  & .form-control:focus {
     outline: none;
```

Comment:
na `outline: none` sa este pozrem, imho nie je spravne to nastavovat.

Reply:
Ono není dobrý udělat něco jako `* { outline: none }`.
Jako vypnout to všude, protože pak nad tím nemáš kontrolu.
Pokud nad tím kontrolu máš, a máš ten focus, hover značený jinak, tak si myslím že to je v pořádku.